ush/python/pyufs/ufswm/ufswm.py

```diff
@@ -0,0 +1,114 @@
+"""
+Module
+------
+
+    ufswm.py
+
+Description
+-----------
+
+    This module contains the base-class object for all Unified
+    Forecast System (UFS) weather-model (WM) applications.
+
+Classes
+-------
+
+    UFSWM(config, model, attr, *args, **kwargs)
+
+        This is the base-class object for all UFS weather-model
+        forecast and related applications; it is a sub-class of Task.
+
+Author(s)
+---------
+
+
+    Henry R. Winterbottom; 28 March 2023
+
+History
+-------
+
+    2023-03-28: Henry Winterbottom -- Initial implementation.
+
+"""
+
+# ----
+
+from pygw.logger import Logger
+from pygw.task import Task
+from pyufs.exceptions import UFSWMError
+
+# ----
+
+
+class UFSWM(Task):
+    """
+    Description
+    -----------
+
+    This is the base-class object for all UFS weather-model forecast
+    and related applications; it is a sub-class of Task.
+
+    Parameters
+    ----------
+
+    config: object
+
+        A Python object containing the application configuration
+        attributes.
+
+    model: str
+
+        A Python string specifying the supported forecast model
+        application.
+
+    app: str
+
+        A Python string specifying the supported forecast model
+        application/type.
+
+    Other Parameters
+    ----------------
+
+    args: Tuple
+
+        A Python tuple of arguments to be based to the base-class
+        constructor.
+
+    kwargs: Dict
+
+        A Python dictionary containing keyword arguments to pass to
+        the base-class constructor.
+
+    Raises
+    ------
+
+    UFSWMError:
+
+        - raised if a specified forecast model is not supported.
+
+    """
+
+    def __init__(self: Task, config: object, model: str, app: str, *args, **kwargs):
```

Comment:
```suggestion
    def __init__(self: Task, config: Dict, model: str, app: str, *args, **kwargs):
```

`config` is a `Dict`

Reply:
This was a mistake. It should be beneath `pyufs/tasks`. I must have missed it during the copy/rename.